types/babel__traverse/index.d.ts

```diff
@@ -90,7 +90,13 @@ export class Scope {
 
     removeData(key: string): void;
 
-    push(opts: any): void;
+    push(opts: {
+        id: object,
+        init?: object,
+        unique?: boolean,
+        _blockHoist?: number,
+        kind?: "var" | "let",
```

Comment:
I think const is also valid in this context?

Reply:
Yup, const works too.